code/datums/autocells/explosion.dm

```diff
@@ -293,6 +293,52 @@ as having entered the turf.
 		create_shrapnel(epicenter, rand(5,9), , ,/datum/ammo/bullet/shrapnel/light/effect/ver1, explosion_cause_data)
 		create_shrapnel(epicenter, rand(5,9), , ,/datum/ammo/bullet/shrapnel/light/effect/ver2, explosion_cause_data)
 
+	// everything below this line is strictly for handling ob camera shakes
+	// it can be modified in the future to handle other explosion types by interacting with power and falloff instead.
+	if(!warhead_kind)
```

Comment:
I think all of this should really be handled in OB code. If you're specifying "warhead_kind" you're looking at something that likely should be handled via object oriented programming. Sorry to have you refactor it again 😶

Reply:
It was initially like that, but I moved it over with the expectation someone down the line would want to integrate it with this proc. I'll move it back.